grpc/src/main/java/com/linecorp/armeria/server/grpc/GrpcServiceBuilder.java

```diff
@@ -494,6 +496,17 @@ public GrpcServiceBuilder exceptionMapping(GrpcStatusFunction statusFunction) {
      * <p>Note that this method and {@link #exceptionMapping(GrpcStatusFunction)} are mutually exclusive.
      */
     public GrpcServiceBuilder addExceptionMapping(Class<? extends Throwable> exceptionType, Status status) {
+        return addExceptionMapping(exceptionType, status, null);
+    }
+
+    /**
+     * Adds the specified exception mapping that maps a {@link Throwable} to a gRPC {@link Status}.
+     * The mapping is used to handle a {@link Throwable} when it is raised.
+     *
+     * <p>Note that this method and {@link #exceptionMapping(GrpcStatusFunction)} are mutually exclusive.
+     */
+    public <T extends Throwable> GrpcServiceBuilder addExceptionMapping(
+            Class<T> exceptionType, Status status, @Nullable BiConsumer<T, Metadata> metadataFunction) {
```

Comment:
What do you think about just accepting a `GrpcStatusFunction` instead of `status` and `metadataFunction`?

Reply:
I agree.

Reply:
Changed to

```java
addExceptionMapping(Class<T> exceptionType, BiFunction<T, Metadata, Status> statusFunction)
```